src/neurotechdevkit/scenarios/_base.py

```diff
@@ -29,6 +27,8 @@
     select_simulation_time_for_steady_state,
 )
 from ._utils import (
+    SliceAxis,
+    Target,
```

Comment:
Do you need to update the corresponding import in `__init__.py` for `Target` to be `from ._utils` as well?

https://github.com/agencyenterprise/neurotechdevkit/blob/6f10b777c3f2a704b81910ab118c8cde9f12d7c6/src/neurotechdevkit/scenarios/__init__.py